core/lib/instrument.c

```diff
@@ -2784,6 +2784,53 @@ dr_get_application_name(void)
 #    endif
 }
 
+dr_error_code_t
+dr_get_error_code(void)
+{
+    dcontext_t *dcontext = get_thread_private_dcontext();
```

Comment:
Hmm, would we ever want to pass in the dcontext to support getting on another thread (also slight perf opt to pass already-obtained dcontext)?  While the last error does seem like an execution thread thing unlikely to involve the non-executing thread are we going to regret it later.  Hmm.  It does seem like there is very unlikely to be a use case that wouldn't involve the currently executing thread.

Reply:
Done.